lib/ansible/module_utils/influxdb.py

```diff
@@ -44,6 +44,7 @@ def influxdb_argument_spec():
             port=dict(default=8086, type='int'),
             username=dict(default='root', type='str', aliases=['login_username']),
             password=dict(default='root', type='str', no_log=True, aliases=['login_password']),
+            database_name=dict(default=None, type='str'),
```

Comment:
InfluxDBClient default value for 'database' parameter is 'None': https://github.com/influxdata/influxdb-python/blob/master/influxdb/client.py#L69.

Reply:
i suggested same fix to lib/ansible/module_utils/influxdb.py
some time ago in PR #31566

Reply:
So, this means if no database is given, database is None. The common args does not have to match the args in the influxdb-python lib.

If we add a `database_name` to the common args, it means that in the module influxdb_user, we can pass a database_name which is not handled at all and does not make sense.

Reply:
that is why database_name is not a common arg

Reply:
Please take a look in init of InfluxDb(): and then on connect_to_influxdb(self):
they both use database_name and it's mandatory in such way.

Reply:
it may be possible that each module will override it inside by under argument_spec.update()by adding database_name

Reply:
now I see.  Seems one change didn't make it to this file. I fixed this by in this way:

~~~diff
- database=self.database_name,
+ database=self.params.get('database_name'),
~~~

Can you add this fix instead?

Reply:
@resmo Done in 0df2d10.